native/core/src/events/news.rs

```diff
@@ -101,50 +98,6 @@ pub struct NewsSlide {
     pub references: Vec<ObjRef>,
 }
 
-impl NewsSlide {
```

Comment:
I think I already mentioned in a previous review that I am not fond of moving APIs that are generic and can be used rather broadly outside the ffi-layer (so, without a `RUNTIME`) out of `core` and into the higher level. I'd rather want the higher level as thin as possible: everything that needs RUNTIME must be there, like things that interact with client, but things that do not need it should be located in the underlying crate. That is especially true for the instantiation of types from this crates that the upper crate ideally doesn't do directly but always uses the helper functions for, as existed here before.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/978/commits/ef091de0739d8d31a253cec981d154fb4d4895ec